lightning/src/ln/channel.rs

```diff
@@ -3031,9 +3054,14 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 			},
 		};
 
+		let closing_tx_max_weight = self.get_closing_transaction_weight(
+			if let Some(oup) = closing_tx.output.get(0) { Some(&oup.script_pubkey) } else { None },
```

Comment:
Why would this ever not have 0/1 outputs?

Reply:
Close outputs can be dust, causing that channel party's balance to be burned to fees at close.